includes/event/event-form-handler.php

```diff
@@ -73,8 +77,10 @@ public function handle( array $form_data ): void {
 				$response_message = esc_html__( 'Failed to delete event.', 'gp-translation-events' );
 				$event_status     = $event->status();
 			} else {
-				$response_message = esc_html__( 'Event deleted successfully.', 'gp-translation-events' );
-				$event_status     = 'deleted';
+				$response_message       = esc_html__( 'Event deleted successfully.', 'gp-translation-events' );
+				$event_status           = 'deleted';
+				$notifications_schedule = new Notifications_Schedule( $this->event_repository );
```

Comment:
I think it would make sense to move the instantiation of `Notication_Schedule` to the constructor, similarly to `Event_Repository`. That we can do `$this->notifications_schedule->delete_scheduled_emails( $event_id );`

Reply:
Done!